fiftyone/zoo/models/manifest-torch.json

```diff
@@ -1862,6 +1862,52 @@
             ],
             "date_added": "2020-12-11 13:45:51"
         },
+        {
+            "base_name": "open-clip",
```

Comment:
Can you please use `"open-clip-torch"` as the name for consistency with how other models are named?